pyproject.toml

```diff
@@ -42,3 +42,27 @@ module-name = "uv"
 python-source = "python"
 strip = true
 include = ["rust-toolchain.toml"]
+
```

Comment:
(Extra newline here, only because it might get stripped at some point and cause a diff.)